libs/components/pages/src/lib/modules/page/page.module.ts

```diff
@@ -10,5 +14,11 @@ import { SkyPageComponent } from './page.component';
   declarations: [SkyPageComponent, SkyPageContentComponent],
   imports: [CommonModule],
   exports: [SkyPageComponent, SkyPageHeaderModule, SkyPageContentComponent],
+  providers: [
+    {
+      provide: SkyMediaQueryService,
+      useExisting: SkyResizeObserverMediaQueryService,
```

Comment:
This should be provided at the component level, not the module level. Otherwise other modules that import this module will get `SkyResizeObserverMediaQueryService` instead of `SkyMediaQueryService`.

Also, I recommend providing and observing in the page content component instead of the page component, since the page content component may have its own margin/padding that needs to be accounted for. Along those same lines, would other layout host components also need to be responsive? I'm mainly thinking of the tab component, since its inner width is going to be less than the page content's inner width.

```

```